allure-pytest-bdd/src/plugin.py

```diff
@@ -32,6 +57,10 @@ def pytest_configure(config):
     if report_dir:
         report_dir = os.path.abspath(report_dir)
 
+        test_helper = AllureTestHelper(config)
+        allure_commons.plugin_manager.register(test_helper)
+        config.add_cleanup(cleanup_factory(test_helper))
+
```

Comment:
Please, register custom marks like allure-pytest does it [here](https://github.com/allure-framework/allure-python/blob/20ba28a086e9097c4d44f9e01c8df8bf85f41d3e/allure-pytest/src/plugin.py#L171-L174)